pandas/_libs/tslib.pyx

```diff
@@ -1148,7 +1148,8 @@ cdef class _Timestamp(datetime):
 
     cpdef datetime to_datetime(_Timestamp self):
         """
-        DEPRECATED: use :meth:`to_pydatetime` instead.
+        .. deprecated:: 0.19.0
+           Use :meth:`to_pydatetime` instead.
```

Comment:
Can you leave this one as is? The thing is that the first sentence of the docstring gets added to the API overview. So the actual "DEPRECATED .. " is then more useful as the directive code (although I am actually not sure what sphinx would put in the api summary table)